www/includes/easyparliament/templates/html/alert/index.php

```diff
@@ -0,0 +1,371 @@
+                      <?php } ?>
+                    </li>
+
+                  <?php if (isset($member_alertsearch)) { ?>
+                    <li>
+                        <form action="<?= $actionurl ?>" method="post">
+                            <input type="hidden" name="t" value="<?= _htmlspecialchars($token) ?>">
+                            <input type="hidden" name="email" value="<?= _htmlspecialchars($email) ?>">
+                            <input type="hidden" name="keyword" value="<?= _htmlspecialchars($member_alertsearch) ?>">
+                            Mentions of [<?= _htmlspecialchars($member_displaysearch) ?>] by <?= $mp_display_text . $member->full_name() ?>
+                            <input type="submit" class="button small" value="Subscribe">
+                        </form>
+                    </li>
+                  <?php } ?>
+
+
+                  <?php if (isset($scottish_alertsearch)) { ?>
+                    <li>
+                        <form action="<?= $actionurl ?>" method="post">
+                            <input type="hidden" name="t" value="<?= _htmlspecialchars($token) ?>">
+                            <input type="hidden" name="email" value="<?= _htmlspecialchars($email) ?>">
+                            <input type="hidden" name="keyword" value="<?= _htmlspecialchars($scottish_alertsearch) ?>">
+                            Mentions of [<?= _htmlspecialchars($member_displaysearch) ?>] by your MSP, <?= $scottish_member->full_name() ?>
+                            <input type="submit" class="button small" value="Subscribe">
+                        </form>
+                    </li>
+                  <?php } ?>
+                </ul>
+              <?php } ?>
+            </div>
+        </div>
+      <?php } ?>
+
+        <div class="alert-section">
+            <div class="alert-section__primary">
+
+              <?php if ($pid) { ?>
+                <h3>
+                    Sign up for an alert when <?= $pid_member->full_name() ?>
+                    (<?= _htmlspecialchars($pid_member->constituency()) ?>)
+                    speaks.
+                </h3>
+              <?php } else if ($keyword) { ?>
+                <h3>
+                    Sign up for an alert when
+                    [<?= _htmlspecialchars($display_keyword) ?>]
+                    is mentioned.
+                </h3>
+              <?php } else { ?>
+                <h3>Request a new TheyWorkForYou email alert</h3>
+              <?php } ?>
+
+                <form action="<?= $actionurl ?>" method="post" class="alert-page-main-inputs">
+                  <?php if (!$email_verified) { ?>
+                    <p>
+                        <input type="email" class="form-control" placeholder="Your email address" name="email" id="email" value="<?= _htmlentities($email) ?>">
+                        <?php if (isset($errors["email"]) && $submitted) { ?>
+                          <p><?= $errors["email"] ?></p>
```

Comment:
This isn't highlighted as an error at all, I left my email address blank and missed the message.